Makefile

```diff
@@ -106,7 +106,10 @@ dev-init: ## pipe ENVs into docker-compose, prevents need of wrapper script
 
 .PHONY: app-tests-dev
 app-tests-dev: ## Run development tests (dev)
-	docker-compose run django ./manage.py test --noinput --keepdb
+	docker-compose run django /bin/bash -c \
+		"coverage run --source='.' ./manage.py test --noinput --keepdb; \
+		coverage html --skip-empty --omit='*/migrations/*.py'; \
+		coverage report --fail-under=65 --skip-empty --omit='*/migrations/*.py'"
```

Comment:
From looking at the coverage report, it might also be good to omit certain settings files that we're somewhat sure are not going to be run because of the environment, such as `securethenews/settings/production.py`. Or possibly all settings files should be ignored, since they're not really being tested as such?